src/cascade/executor/argument_parser.py

```diff
@@ -99,70 +103,135 @@ def exit(self, status=0, message=None):
         raise ArgumentException(message, status)
 
     @staticmethod
-    def _logging_config(args, epiviz_log_dir=EPIVIZ_LOG_DIR, code_log_dir=CODE_LOG_DIR):
+    def _logging_config(args):
         """Configures logging. Command-line arguments ``-v`` and ``-q``
         set both the code log file and what streams to stdout. The ``-v``
         flag turns on debug level, and ``-q`` sets it to info level.
         The math logger will always be set to info level.
 
         Arguments:
-            args (argparse.Namespace): the arguments parsed by self.
-            epiviz_log_dir (Path): Directory into which to put the math log.
-            code_log_dir (Path): Directory into which to put the code log.
+            args (argparse.Namespace): the arguments parsed by self. This
+                must have members ``verbose``, ``quiet``, ``code_log``,
+                ``mvid``, ``epiviz_log``, ``logmod``, and ``modlevel``.
         """
         # Any handlers from from a basicConfig, which we will reconfigure.
         for handler in logging.root.handlers:
             logging.root.removeHandler(handler)
 
         level = logging.INFO - 10 * args.verbose + 10 * args.quiet
-        fmt = "%(levelname)s %(asctime)s %(name)s %(funcName)s: " "%(message)s"
-        datefmt = "%y-%m-%d %H:%M:%S"
-
         # The command-line logging level specifies what goes to stderr.
         root_handler = logging.StreamHandler(sys.stderr)
+        fmt = "%(levelname)s %(asctime)s %(pathname)s:%(lineno)d: %(message)s"
+        datefmt = "%y-%m-%d %H:%M:%S"
         root_handler.setFormatter(logging.Formatter(fmt, datefmt))
+        root_handler.setLevel(level)
         logging.root.addHandler(root_handler)
         logging.root.setLevel(level)
 
+        code_log = BaseArgumentParser._logging_configure_root_log(args.code_log, level)
+        BaseArgumentParser._logging_configure_mathlog(args.mvid, args.epiviz_log)
```

Comment:
I am not sure what's happening here.  We need to set code_log, but not set anything for mathlog, or the one below?

Reply:
Ah, it lacks symmetry. I want to print to the math log where the code log and math log are located. Because of the way functions are separated, the code log location gets passed to the math log, where it prints.